### PR TITLE
[Bug] ExtractIndicatorsFromWordFile: docker run with non-root user

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/17100

## Description
Added export for HOME.

### Error example:
Screen Shot 2019-07-10 at 13 37 27

### Extract on a file that causes the error (error on top, fix on bottom):
image

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests - testing is done manually, because it only fails when user is not admin
- [ ] Code Review

### CODE REVIEW

**Comment by glicht** [comment]
Let's add a unit test

File: TestPlaybooks/playbook-Process_Email_-_Generic_-_Test.yml [hunk 1]
```diff
@@ -304,7 +304,7 @@ tasks:
         simple: "yes"
       unsecure: {}
       url:
-        simple: https://raw.githubusercontent.com/demisto/content/master/TestData/docwithindicators.doc
+        simple: https://raw.githubusercontent.com/demisto/content/master/TestData/docwithindicators
```
Comment (glicht) [inline TestPlaybooks/playbook-Process_Email_-_Generic_-_Test.yml]: Why did you remove the `.doc`. Seems like a bug

File: TestPlaybooks/playbook-Extract_Indicators_From_File_-_test_4_0.yml [hunk 2]
```diff
@@ -275,7 +275,7 @@ tasks:
         simple: "yes"
       unsecure: {}
       url:
-        simple: https://raw.githubusercontent.com/demisto/content/master/TestData/docxwithindicators.docx
+        simple: https://raw.githubusercontent.com/demisto/content/master/TestData/docxwithindicators
```
Comment (glicht) [inline TestPlaybooks/playbook-Extract_Indicators_From_File_-_test_4_0.yml]: Seem like a bug. I think it should be with `.doc` ending

File: TestPlaybooks/playbook-Process_Email_-_Generic_-_Test.yml (hunk 1 above)
Comment (DeanArbel) [reply TestPlaybooks/playbook-Process_Email_-_Generic_-_Test.yml]: this was done by mistake (refactored another item by this name, and this happened automatically). Fixing it

File: TestPlaybooks/playbook-Extract_Indicators_From_File_-_test_4_0.yml (hunk 2 above)
Comment (DeanArbel) [reply TestPlaybooks/playbook-Extract_Indicators_From_File_-_test_4_0.yml]: Done by mistake. see https://github.com/demisto/content/pull/3845#discussion_r303747258
